deploy/crds/upgrade.managed.openshift.io_upgradeconfigs_crd.yaml

```diff
@@ -0,0 +1,225 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: upgradeconfigs.upgrade.managed.openshift.io
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.phase
+    name: status
+    type: string
+  group: upgrade.managed.openshift.io
+  names:
+    kind: UpgradeConfig
+    listKind: UpgradeConfigList
+    plural: upgradeconfigs
+    shortNames:
+    - upgrade
+    singular: upgradeconfig
+  scope: Cluster
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: UpgradeConfig is the Schema for the upgradeconfigs API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: UpgradeConfigSpec defines the desired state of UpgradeConfig
+            and upgrade window and freeze window
+          properties:
+            desired:
+              description: Specify the desired release
+              properties:
+                channel:
+                  description: Channel we gonna use for upgrades
+                  type: string
+                force:
+                  default: false
```

Comment:
When testing, I couldn't create the CRD due to this:
`The CustomResourceDefinition "upgradeconfigs.upgrade.managed.openshift.io" is invalid: spec.validation.openAPIV3Schema.properties[spec].properties[desired].properties[force].de
fault: Forbidden: must not be set`

I tried on a minikube (1.18) and an OSD cluster. After a bit of research I [noticed a comment elsewhere](https://github.com/coreos/prometheus-operator/issues/3122#issuecomment-610419546) noting that defaults are only supported in CRDs using apiextensions v1.